src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -72,7 +72,8 @@ void regmodule_properties(py::module m) {
         .value("ECORE_ONLY", ov::hint::SchedulingCoreType::ECORE_ONLY);
 
     py::enum_<ov::hint::ModelDistributionPolicy>(m_hint, "ModelDistributionPolicy", py::arithmetic())
-        .value("TENSOR_PARALLEL", ov::hint::ModelDistributionPolicy::TENSOR_PARALLEL);
+        .value("TENSOR_PARALLEL", ov::hint::ModelDistributionPolicy::TENSOR_PARALLEL)
+        .value("PIPELINE_PARALLEL", ov::hint::ModelDistributionPolicy::PIPELINE_PARALLEL);
```

Comment:
em.. the pipeline parallel seems confusing, because actually in hetero cases it does not introduce additional parallel I think
have we aligned already on the name here?

Reply:
Yes, we have aligned already.

Reply:
@wangleis , any advice to this?